ztp/du-profile-kube-compare/required/machine-config/enable-cgroup-v1/enable-cgroup-v1.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: config.openshift.io/v1
```

Comment:
In 4.16 this was removed from source-crs

Reply:
removed